Dev-Corner/slides/toolshed.html

```diff
@@ -0,0 +1,861 @@
+                        <script type="text/template">
+                            ## References
+                            - [ToolShed](https://wiki.galaxyproject.org/ToolShed/Tour)
+                            - [Install tools from the ToolShed](https://wiki.galaxyproject.org/Admin/Tools/AddToolFromToolShedTutorial)
+                            - [Publish tools with Planemo](https://planemo.readthedocs.io/en/latest/publishing.html)
+                        </script>
+                    </section>
+                </section>
+                <section>
+                    <section data-markdown >
+                        <script type="text/template">
+                            ### <i class="fa fa-pencil" aria-hidden="true"></i> Hands-on
+                            
+                            <img src="../images/exercise.png" alt="exercise">
+                        </script>
+                    </section>
+                    <section data-markdown >
+                        <script type="text/template">
+                            ### <i class="fa fa-pencil" aria-hidden="true"></i> Hands-on
+                            
+                            #### The goal
+                            Publish your tool (seqtk_seq) in your local toolshed with Planemo
+                        </script>
+                    </section>
+                    <section data-markdown >
+                        <script type="text/template">
+                            ### <i class="fa fa-pencil" aria-hidden="true"></i> Hands-on
+
+                            #### Step 1: Start and configure your toolshed
+                            - Start your toolshed (in galaxy home directory):
+                              - `./run_tool_shed.sh`
+                            - Toolshed available at:
+                              - `http://localhost:9009`
+                        </script>
+                    </section>
+                    <section data-markdown >
+                        <script type="text/template">
+                            ### <i class="fa fa-pencil" aria-hidden="true"></i> Hands-on
+
+                            #### Step 2: Use planemo to publish your tool
+                         </script>
+                    </section>
+                    <section data-markdown >
+                        <script type="text/template">
+                            ### <i class="fa fa-pencil" aria-hidden="true"></i> Hands-on
+
+                            #### Step 2: Use planemo to publish your tool
+                            - `planemo config_init`
+                            - `planemo shed_init`
+                            - `planemo shed_lint`
+                            - `planemo shed_create`
+                        </script>
+                    </section>
+                    <section data-markdown >
+                        <script type="text/template">
+                            ## Answers
```

Comment:
I have removed the "Answers" section. I have added some elements into `aside class="notes" ` tags. Should I create something somewhere else for trainees ?